pom.xml

```diff
@@ -84,6 +84,11 @@
         <artifactId>jenkins-test-harness-tools</artifactId>
         <version>2.2</version>
       </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.main</groupId>
+        <artifactId>jenkins-test-harness</artifactId>
+        <version>1697.v6b_1e34cb_4ee6</version>
+      </dependency>
```

Comment:
This is defined in the `dependencyManagement` of the parent pom, so you can remove the version and instead override https://github.com/jenkinsci/plugin-pom/blob/bd11e866db97ccf017b504d64f4821d897ea8987/pom.xml#L67 for coherence of usage with other plugins

Reply:
given it was released today I highly doubt it :)